popup.html

```diff
@@ -2,123 +2,173 @@
 <head>
 	<title></title>
 	<style>
-
-	* {
-		box-sizing: border-box;
-	}
-	body {
-		width: 391px;
-		color: cornflowerblue;
-	}
-	img {
-		max-width: 100%;
-		height: auto;
-	}
-
-	.psiScore {
-		display:none;
-		text-align: center;
-	}
-
-	.column_2 {
-		float: left;
-		width: 50%;
-	}
-
-	.row:after {
-		content: "";
-		display: table;
-		clear: both;
-	}
-
-	.grades {
-		font-size: 33px;
-		padding: 16px;
-		color: #e43;
-	}
-
-	.score {
-		font-size: 40px;
-		display: block;
-		font-weight: bold;
-	}
-
-	a#hsDebug {
-		width: 35%;
-	}
-
-	a#hsMoveJQueryToFooter {
-		width: 64%;
-	}
-
-	.debugButton {
-		height: 44px;
-		display: inline-block;
-		border: 3px solid #e43;
-		font-size: 14px;
-		text-align: center;
-		text-decoration: none;
-		overflow: hidden;
-		cursor: pointer;
-	}
-
-	.debugButton.graded {
-		overflow: initial;
-		height: initial;
-	}
-
-	#psiScoreRequest {
-		width: 100%
-	}
-
-	#psiScoreRequest.graded {
-		background-image: url(https://cdn2.hubspot.net/hubfs/2359872/Dev_Extension/pagespeed-logo.png);
-		background-size: contain;
-		background-position: center bottom;
-		background-repeat: no-repeat;
-		background-size: 150px;
-	}
-
-	#psiScoreRequest.graded .face-secondary{
-		display: none;
-	}
-
-	.debugButton .face-primary,
-	.debugButton .face-secondary {
-		display: block;
-		padding: 0 22px;
-		line-height: 39px;
-		transition: margin .4s;
-	}
-	.debugButton .face-primary {
-		background-color: #e43;
-		color: #fff;
-	}
-	.debugButton:not(.graded):hover .face-primary {
-		margin-top: -39px;
-	}
-
+/*to keep CSS organized and easy to understand this css is following this method https://css-tricks.com/combining-the-powers-of-sem-and-bio-for-improving-css/
+Do not let fear of learning a new technique prevent you from contributing, others can help convert your css to fit this flow. so pull request even if it doesn't fit this way of doing things. The important thing is that it works, keeping organized and understandable is second. If everyone finds they hate writing CSS this way we can work out another way to do it.
+*/
+/*Settings - variables that do not generate visual styles on their own*/
+:root{
+  --buttonColor:#ff7a59;
+}
+/*Objects - (namespace: o-) are used for design patterns, such as layouts, where items are being arranged rather than decorated. */
+
```

Comment:
Could we get something like `:focus {outline: none;}` in here? 

Looks much better by the way!

Reply:
Yep introducing the focus change now. I made it so if you tab through it will now trigger hoverstate